infrastructure/terragrunt.hcl

```diff
@@ -32,6 +32,10 @@ terraform {
       source  = "hashicorp/azurerm"
       version = ">= 3.32"
     }
+    databricks = {
+      source = "databricks/databricks"
+      version = "1.9.0"
```

Comment:
1.9.1 is latest? is there a reason not to use that?

Reply:
Good point, will update